sdk-workflows/src/main/java/io/dapr/workflows/DaprWorkflowContextImpl.java

```diff
@@ -204,4 +205,12 @@ public void continueAsNew(Object input) {
   public void continueAsNew(Object input, boolean preserveUnprocessedEvents) {
     this.innerContext.continueAsNew(input, preserveUnprocessedEvents);
   }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public UUID newUuid() {
```

Comment:
Should this be `newUUID()` (caps)? That would make it consistent with both the `UUID` type name of the name of the method in the Durable Task SDK.

Reply:
Dapr java sdk has a style check that doesn't allow the consecutive upper-case character in the method name, so I have to make it in this way.